integration-tests/clipper_manager_tests.py

```diff
@@ -8,7 +8,7 @@
 from argparse import ArgumentParser
 cur_dir = os.path.dirname(os.path.abspath(__file__))
 sys.path.insert(0, os.path.abspath('%s/../' % cur_dir))
-import clipper_manager
+from clipper_admin import clipper_manager
```

Comment:
AFAICT `clipper_manager` is only used to access the `Clipper` class so we could directly import `Clipper` here